steely/plugins/btc.py

```diff
@@ -1,33 +1,63 @@
 '''Get the latest Bitcoin price (in USD and EUR)'''
 import requests
 
-__author__ = 'CianLR'
+__author__ = ('CianLR', 'byxor')
 COMMAND = 'btc'
 
 API_URL = 'https://api.coindesk.com/v1/bpi/currentprice.json'
 RESP_TEMPLATE = """Bitcoin Price:
```

Comment:
This header has been removed in prod, may as well remove it here too

Reply:
The __author__ attribute is deprecated now?

Reply:
No. In prod,
```
RESP_TEMPLATE = """
eur{}
dollar{}
```

or something. The `Bitcoin Price:` substring title on the output has been removed.

This was what I questioned before that confused you.